package.json

```diff
@@ -354,6 +354,174 @@
 						}
 					}
 				}
+			},
+			{
+				"type": "reactnativedirect",
+				"label": "React Native Direct",
```

Comment:
Same here. Instead of "Direct", lets use "Hermes". Also we should add "Experimental" to the string .